src/plots/plots.js

```diff
@@ -13,134 +13,32 @@ var d3 = require('d3');
 var isNumeric = require('fast-isnumeric');
 
 var Plotly = require('../plotly');
+var Registry = require('../registry');
 var Lib = require('../lib');
 var Color = require('../components/color');
 
 var plots = module.exports = {};
 
-var modules = plots.modules = {},
-    allTypes = plots.allTypes = [],
-    allCategories = plots.allCategories = {},
-    subplotsRegistry = plots.subplotsRegistry = {},
-    transformsRegistry = plots.transformsRegistry = {};
+// Expose registry methods that used to be on Plots for backward-compatibility
+plots.modules = Registry.modules;
```

Comment:
Important. Ensure backward compatibility in the `Plotly.Plots` props and methods :arrow_double_down: